### DISCUSSION TITLE
Useful resources

### DISCUSSION THREAD

**Discussion by harrycook** [UNMigration/HTCDS#18] [General]
Are there other useful information management or case management resources you would be able to share with the community? Please feel free to share here or get in touch and we can see how we can help make them available!

**Comment by philhb** [comment]
Yes, I think we should collate these and publish them in the guidance - would be very useful.

**Reply by philhb** [reply]
Forthcoming webinar 23rd September for those interested: [Case Management Information Systems for Child Protection and Care: Data for Impact Presents the CMIS Assessment, Planning and Governance Toolkits](https://unc.zoom.us/webinar/register/WN_ge-fW_qETJGYj48zKyfORQ)

**Reply by NoelPadalhin** [reply]
Thank you for the invite Phil. Will check our schedules if we can join this one. Cheers. Keep safe and healthy.
